core/gaspool.go

```diff
@@ -25,6 +25,12 @@ import (
 // in a block. The zero value is a pool with zero gas available.
 type GasPool uint64
 
+// SetGas set an initial value for gaspool
+func (gp *GasPool) SetGas(amount uint64) *GasPool {
```

Comment:
as we have discussed, can remove this API.

Reply:
updated